fe/fe-core/src/main/java/com/starrocks/sql/ast/DescribeStmt.java

```diff
@@ -86,6 +97,14 @@ public DescribeStmt(TableName dbTableName, boolean isAllTables, NodePosition pos
         this.isAllTables = isAllTables;
     }
 
+    public DescribeStmt(Map<String, String> tableFunctionProperties, NodePosition pos) {
+        super(pos);
+        this.dbTableName = null;
+        this.totalRows = new LinkedList<>();
+        this.isTableFunctionTable = true;
```

Comment:
We will design more `DESC xxx` in the future, like `DESC credential`, `DESC file format`, besides table, mv, and files table function.
So, it's recommended to use desc_object_type ti distinguish the object.

Reply:
`DescribeStmt` is only for `describe table`.
`files()` is a table function table, so i put it here.